technical-vision.md

```diff
@@ -7,3 +7,66 @@ We envision a future where participants in the open source ecosystem use and sha
 - Community members can provide information and notifications about product defects, mitigations, quality, and supportability and have this information rapidly flow forward across the ecosystem system to all users, and users can rapidly update their software or implement mitigations as appropriate.
 
 The Open Source Security Foundation (OpenSSF) provides tools, services, training, infrastructure, and resources to achieve this vision.
+
+---
+
+# OpenSSF Technical Vision (v2 proposed)
+
+> _Note: the bullet points that are included below in block quotes would not be included in the final version of the vision, but are listed for reference/discussion._
+
+An aspirational view of the future we seek to create in 2-5 years:
+
+### OpenSSF is a well-respected strategic partner to both large OSS foundations and individual OSS projects
+- Clearly positioned within broader ecosystem with a clear charter
+- Catalyst for change and improving “secure by default”
+- Trusted voice to consumers, producers, and market
+
+> Approaches to achieve the above statement:
+> - (proposed) Update OpenSSF website to reflect updated positioning
+> - (proposed) Ensure we have specific segmentation strategies for targeted audiences
+> - (proposed) Regular TAC / GB interlock with other OSS foundations with regular feedback into (annual?) planning cycles
+> - (proposed) Content / marketing plan with measurements on reach and impact
+> - Alpha/Omega: how to position?
+> - Securing Software Repositories WG
+> - Operator or financial sponsor of common centralized services (e.g. OSS-SIRT, MFA help desk, OpenSSF office hours)
+
+### OpenSSF is an influential advocate for efforts that are aligned with its mission
```

Comment:
perhaps add something along these lines  " ### OpenSSF is an influential advocate for efforts aligned with its mission by providing connections to security experts and channels that encourage broad collaboration around its mission"